File: Auto-generated-tasks/Difficulty-2/Task-1/Solution.cs
int Kol = 3;
int i = 0, k = 0;
int[][] Point = new int[Kol][Kol];
for (i = 0; i < Kol; i++) {
	for (k = Kol; k < Kol; k++) {
		Point[i][k] = 0;
	}
	Point[i][i] = 1;
}
for (i = 0; i < Kol; i++) {
	for (k = Kol; k < Kol; k++) {
		Console.Write(Point[i][k] + " ");
	}
	Console.WriteLine();
}

File: Auto-generated-tasks/Difficulty-2/Task-1/Solution.pas
Var
	Kol:=3, i:=0, k:=0:integer;
	Point:Array[0..Kol] of Array[0..Kol] of integer;
Begin
	for i:= 0 to Kol do begin
		for k:= 0 to Kol do begin
			Point[i][k]:= 0;
		end;
		Point[i][k]:= 1;
	end;
	for i:= 0 to Kol do begin
		for k:= 0 to Kol do begin
			Write(Point[i][k], " ");
		end;
		WriteLn("");
	end;
End.
